book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [ ] Kindly make sure the rentals have getters and setters, you can add the `rentals` to the `attr_accessor` instead of just `attr_reader`